### PR TITLE
.AspNetCore: adjusting README to current code usage

### PR DESCRIPTION
**Pull request by valbers**
This is a simple PR for adjusting to README to reflect the current state of the code.

### CODE REVIEW

File: src/FSharp.Data.GraphQL.Server.AspNetCore/README.md
Comment (xperiandri) [inline src/FSharp.Data.GraphQL.Server.AspNetCore/README.md]: Maybe as a single line?
``` suggestion
            .UseGiraffe (HttpHandlers.handleGraphQL<Root>)
```

**Comment by valbers** [comment]
@xperiandri waiting for your review approval.

**Review by xperiandri — APPROVED** [review]

**Pull request merged**
